### ISSUE TITLE
Threads are skipped during scrape when Slack client is set to a language other than English

### ISSUE THREAD

**Issue by williamdes** [iulspop/slack-web-scraper#1]
```
Error scraping post. Skipping this post. Index: 13. Error: waiting for selector `[data-qa="slack_kit_list"].c-virtual_list__scroll_container[role="list"][aria-label^="Thread"]` failed: timeout 30000ms exceeded.
Error scraping post. Skipping this post. Index: 35. Error: waiting for selector `[data-qa="slack_kit_list"].c-virtual_list__scroll_container[role="list"][aria-label^="Thread"]` failed: timeout 30000ms exceeded.
Error scraping post. Skipping this post. Index: 39. Error: waiting for selector `[data-qa="slack_kit_list"].c-virtual_list__scroll_container[role="list"][aria-label^="Thread"]` failed: timeout 30000ms exceeded.
```

That said I do not care about thread contents for now, so that's okay.
Maybe in the README the wording should indicate that ENV `CONVERSATION_NAMES=[]` will stop the thread parsing.
I had commented the line and so it was parsing threads by default

Thank you so much for this project !! 🎉

**Comment by williamdes** [comment]
oh okay it does not work because `aria-label` is a translated value

**Comment by iulspop** [comment]
Hi @williamdes, thank you for bringing this problem to my attention! I hadn't considered scraping threads would break if the Slack client used another language than English. Would temporarily changing the Slack client language to English for the scrape be a good solution for you?

**Comment by williamdes** [comment]
> Hi @williamdes, thank you for bringing this problem to my attention! I hadn't considered scraping threads would break if the Slack client used another language than English. Would temporarily changing the Slack client language to English for the scrape be a good solution for you?

I will have to try that but for now changing the strings in the code is an easier option ^^

**Comment by iulspop** [comment]
That works too! I'll get around to improving the error message in case anyone runs into this again, then I'll close this issue.

**Comment by williamdes** [comment]
> That works too! I'll get around to improving the error message in case anyone runs into this again, then I'll close this issue.

Or make an ENV as a workaround ?

**Comment by iulspop** [comment]
Finally, I decided to go with a clearer error message when the thread fails and a note in the README to change the language to English for the scrape. I prefer not to add too many config options to the `.env` file. Also, I don't want to open the project to support multiple languages at the moment. The parsing is tricky enough with one language to account for.
